lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2035,6 +2043,25 @@ def deploy_vm(self):
                     relospec.host = self.select_host()
                 relospec.datastore = datastore
 
+                # Convert disk present in template if is set
+                if self.params['convert']:
+                    if self.params['convert'] not in ['thin', 'thick', 'eagerzeroedthick']:
```

Comment:
With `choices` you don't need this check.

Reply:
Ok i change this thanks